src/components/markdown_editor/markdown_editor_toolbar.tsx

```diff
@@ -107,6 +107,8 @@ export const EuiMarkdownEditorToolbar = forwardRef<
 
     const isPreviewing = viewMode === MODE_VIEWING;
 
+    const buttonIsDisabled = isPreviewing || readOnly;
```

Comment:
Nit pick naming:
```suggestion
    const editingIsDisabled = isPreviewing || readOnly;
```

or changing it to a positive const name so that it's not a double negative:

```suggestion
    const isEditable = !isPreviewing && !readOnly;
```